SwiftLeeds/Views/About/SponsorGridView.swift

```diff
@@ -0,0 +1,69 @@
+//
+//  SponsorGridView.swift
+//  SwiftLeeds
+//
+//  Created by Alex Logan on 01/07/2022.
+//
+
+import SwiftUI
+
+struct SponsorGridView: View {
+    @Environment(\.openURL) var openURL
+    let goldSponsors = Sponsor.platinum
+
+    var body: some View {
+        VStack(spacing: Padding.cellGap) {
+            sectionHeader(text: "Platinum Sponsors")
+            contentTile(for: Sponsor.codemagic)
```

Comment:
Can this loop through platinum sponsors rather than hard coding them here?

Reply:
Not particularly concerned on this as theyre placeholders for API values.